src/intl/Core__Intl__Common.res

```diff
@@ -1,4 +1,4 @@
-type localeMatcher = [#lookup | @as("best fit") #bestFit]
+type localeMatcher = [#lookup | #bestFit]
```

Comment:
This still seems wrong. It should be `#"best fit"`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#localematcher